kv/checker.go

```diff
@@ -44,8 +44,9 @@ func (d RequestTypeSupportedChecker) supportExpr(exprType tipb.ExprType) bool {
 		tipb.ExprType_Float32, tipb.ExprType_Float64, tipb.ExprType_ColumnRef, tipb.ExprType_MysqlEnum:
 		return true
 	// aggregate functions.
+	// NOTE: tipb.ExprType_GroupConcat is only supported to tiflash, So checking it for tikv outside.
```

Comment:
Some grammatical error. My suggestion is : .... supported by TiFlash, so we check the TiKV case outside.